fractor-doc-generator/tests/Printer/DirectoryToMarkdownPrinter/DirectoryToMarkdownPrinterTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorDocGenerator\Tests\Printer\DirectoryToMarkdownPrinter;
+
+use a9f\FractorDocGenerator\Printer\DirectoryToMarkdownPrinter;
+use a9f\FractorDocGenerator\Tests\AbstractTestCase;
+use PHPUnit\Framework\Attributes\DataProvider;
+
+final class DirectoryToMarkdownPrinterTest extends AbstractTestCase
+{
+    private DirectoryToMarkdownPrinter $directoryToMarkdownPrinter;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->directoryToMarkdownPrinter = $this->getService(DirectoryToMarkdownPrinter::class);
+    }
+
+    #[DataProvider('provideData')]
+    public function test(string $directory, string $expectedFile): void
+    {
+        $fileContent = $this->directoryToMarkdownPrinter->print(__DIR__, [$directory]);
+
+        self::assertStringEqualsFile($expectedFile, $fileContent, $directory);
+    }
+
+    public static function provideData(): \Iterator
+    {
+        yield [__DIR__ . '/Fixture/Fractor/Standard', __DIR__ . '/Expected/fractor/fractor_content.md'];
```

Comment:
I would place the expected output file as close as possible to the Standard folder, to make their relation clearer, e.g. in `…/Fixture/Fractor/Standard_ExpectedOutput.md`